app/views/publisher_information/how_to_use_publisher.govspeak.md

```diff
@@ -1,19 +1,17 @@
-Content publisher is a new way of publishing content on GOV.UK. It is designed to make it 
+Content Publisher is a new way of publishing content on GOV.UK. It is designed to make it 
 easier to publish useful content and to help you use data to manage that content.
 
-GDS are now beta testing Content publisher with government publishers. The beta version will have limited functionality at first. We are adding and improving features. 
+We are now beta testing Content Publisher with government publishers. The beta version will have limited functionality at first. We are adding and improving features. 
 
-Read more about what Content publish can and cannot do.
-
-This document is for people who need to publish content to help them know what they should do with the new tool.
+Read more about [what Content Publisher can and cannot do](/beta-capabilities).
 
 ##How to create content
 
-Content publisher can be used to create news stories and press releases. Other document types still need to be created in Whitehall publisher.
+Content Publisher can be used to create news stories and press releases. Other document types still need to be created in Whitehall publisher.
 
 You should always consider if content really needs to be published on GOV.UK. Unnecessary content makes it harder for users to find what the need.
 
-Read guidance on what to publish on GOV.UK.
+Read [guidance on what to publish on GOV.UK](documents/publishing-guidance).
```

Comment:
I think we have a syntax to make this open in a separate tab - should we do that for the links on this page?

Reply:
My understanding is that we use that syntax for pages where we could lose input data in case users click a link - as this is on a guidance page we shouldn't open the link in a new tab.